lib/engine/g_1828/shell.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Engine
+  module G1828
+    class Shell
+      attr_accessor :name, :trains, :system
```

Comment:
trains should be a reader right? do you actually need to make a new array?

Reply:
what exactly is a shell?

Reply:
Yes, trains should be a reader